Tools/docker_run.sh

```diff
@@ -1,67 +1,22 @@
-#! /bin/bash
+#!/bin/bash
 
-if [ -z ${PX4_DOCKER_REPO+x} ]; then
-	echo "guessing PX4_DOCKER_REPO based on input";
-	if [[ $@ =~ .*px4_fmu.* ]]; then
-		# nuttx-px4fmu-v{1,2,3,4,5}
-		PX4_DOCKER_REPO="px4io/px4-dev-nuttx-focal:2021-09-08"
-	elif [[ $@ =~ .*navio2.* ]] || [[ $@ =~ .*raspberry.* ]] || [[ $@ =~ .*beaglebone.* ]] || [[ $@ =~ .*pilotpi.default ]]; then
-		# beaglebone_blue_default, emlid_navio2_default, px4_raspberrypi_default, scumaker_pilotpi_default
-		PX4_DOCKER_REPO="px4io/px4-dev-armhf:2021-08-18"
-	elif [[ $@ =~ .*pilotpi.arm64 ]]; then
-		# scumaker_pilotpi_arm64
-		PX4_DOCKER_REPO="px4io/px4-dev-aarch64:latest"
-	elif [[ $@ =~ .*navio2.* ]] || [[ $@ =~ .*raspberry.* ]] || [[ $@ =~ .*bebop.* ]]; then
-		# posix_rpi_cross, posix_bebop_default
-		PX4_DOCKER_REPO="px4io/px4-dev-armhf:2021-08-18"
-	elif [[ $@ =~ .*clang.* ]] || [[ $@ =~ .*scan-build.* ]]; then
-		# clang tools
-		PX4_DOCKER_REPO="px4io/px4-dev-clang:2021-02-04"
-	elif [[ $@ =~ .*tests* ]]; then
-		# run all tests with simulation
-		PX4_DOCKER_REPO="px4io/px4-dev-simulation-bionic:2021-12-11"
-	fi
+if [[ -z "${DOCKER_TAG}" ]]; then
+  TAG_NAME="latest"
 else
-	echo "PX4_DOCKER_REPO is set to '$PX4_DOCKER_REPO'";
+  TAG_NAME="${DOCKER_TAG}"
 fi
 
-# otherwise default to nuttx
-if [ -z ${PX4_DOCKER_REPO+x} ]; then
-	PX4_DOCKER_REPO="px4io/px4-dev-nuttx-focal:2021-09-08"
-fi
-
-# docker hygiene
-
-#Delete all stopped containers (including data-only containers)
-#docker rm $(docker ps -a -q)
+PX4_DOCKER_REPO="px4io/px4-dev:$TAG_NAME"
 
-#Delete all 'untagged/dangling' (<none>) images
-#docker rmi $(docker images -q -f dangling=true)
-
-echo "PX4_DOCKER_REPO: $PX4_DOCKER_REPO";
+echo "[docker_run.sh]: Running '$PX4_DOCKER_REPO'"
 
 PWD=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 SRC_DIR=$PWD/../
 
-CCACHE_DIR=${HOME}/.ccache
-mkdir -p "${CCACHE_DIR}"
-
 docker run -it --rm -w "${SRC_DIR}" \
-	--env=AWS_ACCESS_KEY_ID \
-	--env=AWS_SECRET_ACCESS_KEY \
-	--env=BRANCH_NAME \
-	--env=CCACHE_DIR="${CCACHE_DIR}" \
-	--env=CI \
```

Comment:
I'm not sure if we want to drop all of these.

Reply:
you are right I removed them for ease of debugging but will bring them back